src/unit_tests/wazuh_db/test_wdb_global.c

```diff
@@ -7370,6 +7370,78 @@ void test_wdb_global_set_agent_groups_set_group_ctx_error(void **state) {
     __real_cJSON_Delete(j_group_array);
 }
 
+void test_wdb_global_set_agent_groups_sync_status_invalid_stmt(){
+    expect_value(__wrap_wdb_init_stmt_in_cache, statement_index, WDB_STMT_GLOBAL_GROUP_SYNC_SET);
+    will_return(__wrap_wdb_init_stmt_in_cache, NULL);
+
+    assert_int_equal(OS_INVALID, wdb_global_set_agent_groups_sync_status((wdb_t *)0xDEADBEEF, 1, "test"));
+}
+
+void test_wdb_global_set_agent_groups_sync_status_bad_bind_sync(){
+    wdb_t wdb = {0};
+    wdb.id = "agent001";
+
+    char sync[] = "test";
+
+    expect_value(__wrap_wdb_init_stmt_in_cache, statement_index, WDB_STMT_GLOBAL_GROUP_SYNC_SET);
+    will_return(__wrap_wdb_init_stmt_in_cache, 1);
```

Comment:
Can we add a cast, specially for understanding of that **1** is only a test value, like:
`will_return(__wrap_wdb_init_stmt_in_cache, (sqlite3_stmt*)1); //Returning any value`